common/changes/@itwin/core-quantity/WZ-move-scheduleScriptReference-internal_2025-02-04-22-03.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/core-quantity",
+      "comment": "moved scheduleScriptReference internal",
```

```suggestion
      "comment": "",
```

Just reacting to change in core-frontend, no need for change log.